components/hab/src/command/bldr/job/start.rs

```diff
@@ -65,7 +65,7 @@ pub fn start(
 
     ui.status(
         Status::Created,
-        format!("build job, the id is {}", id),
+        format!("build job. The id is {}.", id),
```

Comment:
Minor - I'd remove the period at the end - it's a tiny bit easier to copy the id if there is no period :)

Reply:
Hehe, ok I'll remove it